source/renderer/app/components/assets/AssetSettingsDialog.tsx

```diff
@@ -209,10 +213,12 @@ class AssetSettingsDialog extends Component<Props, State> {
                       recommendedDecimals,
                     })}
                   >
-                    <SVGInline
-                      className={styles.warningIcon}
-                      svg={warningIcon}
-                    />
+                    <span data-testid="warning-icon">
```

Comment:
I don't think we have this rule set anywhere, but I believe we have been using camel case for testIds.